haystack/components/rankers/transformers_similarity.py

```diff
@@ -43,6 +43,9 @@ def __init__(
         top_k: int = 10,
         meta_fields_to_embed: Optional[List[str]] = None,
         embedding_separator: str = "\n",
+        scale_score: bool = True,
```

Comment:
I don't remember what did we agreed on regarding these optionals in init method?

Reply:
I'm not sure, was there a discussion around variables like `scale_score` in the init method? Also we have other variables in the init method that are optional like `meta_fields_to_embed` which is also used in our Embedders.

Reply:
Ok, you're right, nevermind, I'll jot down that we correct these inconsistencies